vrs/vrs_schema_renaming_script_temp.py

```diff
@@ -0,0 +1,75 @@
+"""
+ Script to update column names from GA4GH VRS-Python Branch 0.8.1 to 0.8.2 convention
+ See: https://github.com/ga4gh/vrs-python/releases/tag/0.8.2
+ Note that this script will be run on 05-04-23 or 05-05-23 as a temporary fix
+ These changes will be permenantly added to the script which adds the VRS Annotations in a later update
+ Note that this code may not be fully productionized or up-to-date with the rest of the team's code standards 
+"""
+
+import argparse
+import hail as hl
+
+
+def main(args):
+
+    import_path = args.import_ht
+    ht = hl.read_table(import_path)
+    ht = ht.sample(0.01)
+
+    # Code which actually changes naming schema from VRS-Python Branch 0.8.1 to 0.8.2
+
+    ht = ht.annotate(
+        info=ht.info.annotate(
+            vrs=ht.info.vrs.rename(
+                {
+                    "VRS_Allele": "VRS_Allele_IDs",
+                    "VRS_Start": "VRS_Starts",
+                    "VRS_End": "VRS_Ends",
+                    "VRS_Alt": "VRS_States",
+                }
+            )
+        )
+    )
+
+    ht_reformat = ht.annotate(
+        info=ht.info.annotate(
+            vrs=ht.info.vrs.annotate(
+                VRS_Allele_IDs=ht.info.vrs.VRS_Allele_IDs.split(","),
+                VRS_Starts=[
+                    hl.int(ht.info.vrs.VRS_Starts.split(",")[0]),
+                    hl.int(ht.info.vrs.VRS_Starts.split(",")[1]),
+                ],
+                VRS_Ends=[
+                    hl.int(ht.info.vrs.VRS_Ends.split(",")[0]),
+                    hl.int(ht.info.vrs.VRS_Ends.split(",")[1]),
+                ],
```

Comment:
```suggestion
                VRS_Ends=ht.info.vrs.VRS_Ends.split(',').map(lambda x: hl.int(x)),
```

Reply:
double checked this and the prior fix in a notebook to the side, and I can confirm that they do both produce results with array<int32> type